### ISSUE TITLE
documented *_command attrs for service resource are misleading

### ISSUE THREAD

**Issue by jblaine** [chef-boneyard/chef-web-docs-2016#449]
Originally here: https://github.com/chef/chef/issues/2474

The short of the issue is that the documentation mentions start_command, stop_command, et al... for use with the 'service' resource, but really they're not to be used (from what I gather, not to be used unless you are using undocumented `Chef::Provider::Service::Simple` provider).

**Comment by jamescott** [comment]
This is an interesting issue. I talked with some of the engineers and this is what was decided:

https://github.com/chef/chef-docs/commit/ec432e9eef53e50cc87a8a01210479a45a7db508

Here's an example of a custom provider that sub-classes the Simple provider:

https://gitorious.org/openbsd-ports-wip/mainline/source/2ef826a8d731b9f8b54f5b7da4395783afefddef:sysutils/ruby-chef/chef/patches/patch-lib_chef_provider_service_openbsd_rb

and here's a provider within Chef itself (systemd) that is also sub-classed:

https://github.com/chef/chef/blob/f8df84e6748e7dd71343fd7844255aa37dbfab78/lib/chef/provider/service/systemd.rb

Hope this helps!

james

**Comment by jblaine** [comment]
I can swallow that, but it leaves an open question about what exactly all of the service resource's *_command attributes are documented publicly for. I'm confused about the use case for them in light of the guidance I think I'm hearing that says not to use them with any of the existing providers.

Thanks

**Comment by jamescott** [comment]
Right. I'm still thinking about if/how the docs should address that aspect.
